cli/src/config/misc.toml

```diff
@@ -6,7 +6,7 @@ tree-level-conflicts = true
 
 [ui]
 paginate = "auto"
-pager = { command = ["less", "-FRX"], env = { LESSCHARSET = "utf-8" } }
+pager = ":builtin"
```

Comment:
I think we will want to leave this unchanged until `minus` works as well as `less -FRX`. If the current state of `minus` means that it's already an improvement on Windows, we can start by enabling it only there. What do you think?

Reply:
Yeah, I'm fine with that. Should we keep using my fork (which adds a -X equivalent) or just use minus upstream until we get it up to parity?

Reply:
I'd prefer to use upstream, but I don't feel strongly. I hope upstream will accept your patch somewhat soon. If that doesn't happen, we can switch over to your fork.

Reply:
That seems good. Swapped back to upstream and restored default to less -FRX (except on Windows)